frontend/src/layouts/header-layout.tsx

```diff
@@ -0,0 +1,54 @@
+import bg from '@/assets/wanderlustbg.webp';
+import ThemeToggle from '@/components/theme-toggle-button';
+import AddIcon from '@/assets/svg/add-icon-white.svg';
+import { useNavigate } from 'react-router-dom';
+function header() {
+    const navigate = useNavigate();
+
+    return (
+    <div
+        style={{ backgroundImage: `url(${bg})` }}
+        className="relative -mt-2 h-[460px] bg-cover bg-fixed bg-center"
```

Comment:
Can you kindly refactor this to use bg-[url] kinda format, the tailwind supported one

Reply:
Done everything.

Reply:
Created a PR with new branch.